docs/release_notes/5.0.24-21.md

```diff
@@ -0,0 +1,22 @@
+# Percona Server for MongoDB 5.0.24-21 (2024-02-01)
+
+[Installation](../install/index.md){.md-button}
+
+Percona Server for MongoDB 5.0.24-21 is an enhanced, source-available, and highly-scalable database that is a
+fully-compatible, drop-in replacement for [MongoDB 5.0.24 Community Edition](https://www.mongodb.com/docs/manual/release-notes/5.0/#5.0.24---jan-18--2024).
+
+It supports MongoDB 5.0.24 protocols and drivers.
+
+## Release Highlights
+
+The bug fixes, provided by MongoDB and included in Percona Server for MongoDB, are the following:
+
+* [SERVER-50792](https://jira.mongodb.org/browse/SERVER-50792) - Improved shard key index error messages by adding detailed information about an invalid index.
+* [SERVER-77506](https://jira.mongodb.org/browse/SERVER-77506) - Fixed the issue with data and ShardVersion mismatch on sharded multi-document transactions by exposing the maxValidAfter timestamp alongside the shardVersion
+* [SERVER-81878](https://jira.mongodb.org/browse/SERVER-81878) - Improved the startupRecoveryForRestore behavior by allowing writes in read-only mode from ident reaper
```

Comment:
SERVER-81878 should be excluded from the list because we reverted this fix (see SERVER-84432)

Reply:
Excluded